### ISSUE TITLE
Database-Distro-specific-examples documentation improvement

### ISSUE THREAD

**Issue by piniguez** [open-mpi/orcm#132]
I was trying the documentation to setup the database but found a couple of issues which were a headage to solve them:

Issue 1 ==>  
      ORCM specific database creation script ask for getting scripts from:
<pre>
https://github.com/open-mpi/orcm/tree/v0.5.0/contrib/database
</pre>

this is wrong and proper path to get them is from:

<pre>
     orcm/build/contrib/database
</pre>

Issue 2 ==> 
Through the setup steps it asks to configure orcmuser without password and doesn't contain instructions to set the password.

Client access setup section (notice trust values instead of password) ==>
<pre>
vi /var/lib/pgsql/9.3/data/pg_hba.conf # Add "host all all 127.0.0.1/32 trust" and "host all all ::1/128 trust
</pre>
but at the end it uses orcmuser with password
<pre>
RUN ORCM section (notice password=orc)==>
orcmd --omca sensor heartbeat,coretemp --omca db_base_verbose 100 --omca db_odbc_dsn orcmdb_psql --omca db_odbc_user orcmuser:orc --omca db_odbc_table data_sample 
</pre>

also check it missing to tell users how to set orcmuser password, that is,
<pre>
psql --user=orcmuser --dbname=orcmdb
             ALTER USER orcmuser WITH PASSWORD 'orc'
             ;
             \q
</pre>

**Comment by vpedabal** [comment]
Is this fixed?

**Comment by avilcheslopez** [comment]
Updated the distro-specific examples wiki page to fix these issues.

For the "trust" vs. "password" configuration...  I left the instructions as they were but added a note saying that using the "trust" authentication method is okay as an initial setup, but once everything is up and running, "password" authentication should be used.  There's a section at the end that explains how to set up the password.  I also updated that section to show how to set the password for the `orcmuser` database user.
